controllers/admin_controller.js

```diff
@@ -30,20 +31,36 @@ async function handleAddUser(req, res) {
     }
 }
 
-async function handleShowUser(req, res) {
-    const showAllUser = await dbHandler.showAll();
+async function handleShowAllUsers(req, res) {
+    const showAllUser = await User.find({});
```

Comment:
`showAllUser` variable should describe the variable value, so in this case - 
User.find return **all users** means the variables name should simply be `users` :)